### PR TITLE
Make Header / Navbar follow the same width as body

### PR DESCRIPTION
**Pull request by hernandev**
Before:

After:

### CODE REVIEW

**Comment by hernandev** [comment]
Sorry @mike-koch did not saw you already sent #6 to fix this, the code is slightly different but the result is the same.

File: github-wide.css [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 @-moz-document domain("github.com") {
 
-.container {
+.container-lg {
+  max-width: 80% !important;
```
Comment (cfoellmann) [inline github-wide.css:6]: 80% is not "full-screen" for me.
Why not 100% to use the full width?

**Comment by nickcampbell18** [comment]
Just came here to find this, anything I can help with to get this merged?

File: github-wide.css (hunk 1 above)
Comment (nickcampbell18) [inline github-wide.css:6]: I think you can also do this with `max-width: none!important`.

**Review by cfoellmann — CHANGES_REQUESTED** [review]
Please change
https://github.com/hernandev/github-wide/blob/05379c95020a002480067791b6a67d95d7fba09a/github-wide.css#L6
to `max-width: none !important;`

**Comment by cfoellmann** [comment]
@nickcampbell18 I hope @hernandev will see my change request soon. Otherwise I will merge in 2 or 3 days.

**Pull request merged**